Automattic/wp-calypso
# Jetpack Error UX: Show error banner in Plugins showcase

See https://github.com/Automattic/dotcom-forge/issues/3394

## Proposed Changes

Shows an error notice on the Plugins showcase when Jetpack is unable to connect to the site:

<img width="1571" alt="image" src="https://github.com/Automattic/wp-calypso/assets/36432/9f611f43-2641-45bb-a08d-db0218130710">

Triggering logic was introduced in https://github.com/Automattic/wp-calypso/pull/79965

## Testing Instructions

1. Create a new Business site and take it Atomic.
2. Add some PHP code to `~/htdocs/wp-content/mu-plugins/local.php` that causes a fatal error.
3. Navigate to Plugins in Calypso.
4. Verify the notice appears as expected.
5. Fix your fatal error.
6. Reload the Plugins view.
7. Verify the notice no longer appears.

Flagging @Automattic/bespin  and @Automattic/explorers  for review in case there are other aspects that need to be tested/considered.
